app/game-session.coffee

```diff
@@ -1,6 +1,6 @@
 {nx} = require 'nexus-node'
 
-RoundPhase    = require 'cssqd-shared/models/round-phase'
+RoundPhase    = require '../shared/models/round-phase'
```

Comment:
why do you like this require method more? @kirilknysh have done great job [here](https://github.com/DaQuirm/css-quickdraw-redux/commit/f30837a78bdd18a846e2c31e5ef250294e39e7b2) :smirk_cat: